### DISCUSSION TITLE
Mode with dispatcher window only

### DISCUSSION THREAD

**Discussion by denglihong2007** [perpetualKid/FreeTrainSimulator#115] [Ideas]
As a player who often works as a dispatcher but doesn't drive trains, my low configuration computer gets stuck when scheduling. I wonder if it is possible to create a mode with no 3D game interface but only a scheduling interface, so as to shield the 3D interface on our computer and make the work of the dispatching work better. thank you.

**Comment by perpetualKid** [comment]
it's planned to have a standalone dispatcher, but will take some more time as few other things need to be updated before - and resources are limited. i.e. the multiplayer code needs some updates, and dispatcher would need an own instance of Simulator (where it currently depends on the 3D Viewer)
